### PR TITLE
adds coveralls.io checks

### LINKED ISSUE #14: Suggestion: Coveralls
I suggest you to setup Coveralls: https://coveralls.io/

With this tools, you will get PR status for coverage.

Example:

image

### PR DESCRIPTION
**Pull request by Algatux**
closes #14

### CODE REVIEW

**Comment by Algatux** [comment]
@Soullivaneuh any suggestion ?

File: .travis.yml [hunk 1]
```diff
@@ -12,6 +9,8 @@ matrix:
   fast_finish: true
   include:
     - php: '7.0'
+      env: DO_COVERAGE="y"
```
Comment (soullivaneuh) [inline .travis.yml]: The coverage should be done on every request.

Some BC trick can be coverage on some builds only.

Coveralls merges all result to have the total coverage.

File: .coveralls.yml [hunk 2]
```diff
@@ -0,0 +1,2 @@
+service_name: travis-ci
```
Comment (soullivaneuh) [inline .coveralls.yml:1]: For me this file is not needed. All my projects work without.

**Comment by soullivaneuh** [comment]
You can also take a look to my integration for comparison: https://github.com/Soullivaneuh/php-cs-fixer-styleci-bridge

**Pull request merged**